include/seqan3/core/concept/cereal.hpp

```diff
@@ -130,6 +131,49 @@ concept bool cereal_text_archive_concept = false;
 #endif
 //!\endcond
 
+/*!\interface seqan3::cerealisable_concept <>
+ * \ingroup core
+ * \brief Is a given type serialisable with cereal?
```

Comment:
We don't use questions anywhere else, please write isntead something like "Specifies the requirements for types that are serialisable via Cereal."